addons/mass_mailing/views/snippets_themes.xml

```diff
@@ -959,7 +959,7 @@
     <!-- COLOR, BORDER | .s_three_columns | .s_comparisons -->
     <div data-js="Box"
          data-selector=".s_three_columns .row > div, .s_comparisons .row > div"
-         data-target=".card">
+         data-target=".card-body">
         <we-colorpicker string="Colors"
```

Comment:
We could also change this to "Background Color" imo :)

Reply:
Good, I changed it, thanks.